tiny_dnn/activations/leaky_relu_layer.h

```diff
@@ -0,0 +1,42 @@
+/*
+    Copyright (c) 2017, Taiga Nomi
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+#include "tiny_dnn/activations/activation_layer.h"
+#include "tiny_dnn/layers/layer.h"
+
+namespace tiny_dnn {
+
+class leaky_relu_layer : public activation_layer {
```

Comment:
I will do that in separate PR. Current PR just replicated classes from **tiny_dnn/activations/activation_function.h** as layers.